lib/common/login.js

```diff
@@ -35,16 +35,16 @@ function login (clientName, options) {
   options = options || {};
   return auth0ClientFactory.getClientIdByName(clientName).then(clientId => {
     if (options.username) {
-      const loginProvider = new UsernameLoginProvider(clientId, clientName);
-      return loginProvider.login(options.username, options.password);
+      const loginProvider = new UsernameLoginProvider(clientId);
+      return loginProvider.login(options.username === true ? null : options.username, options.password);
```

Comment:
I think that `!!options.username || null` would achieve the same thing as this ternary. But perhaps the ternary is more legible and more explicit? /shrug